sources/btc_math.move

```diff
@@ -38,6 +39,47 @@ public fun btc_hash(data: vector<u8>): vector<u8> {
     return second_hash
 }
 
+
+fun compact_size_offset(start_byte: u8): u64 {
+    if (start_byte <= 0xfc) {
+        return 0
+    };
+    if (start_byte == 0xfd) {
+        return 2
+    };
+    if (start_byte == 0xfe) {
+        return 4
+    };
+    // 0xff
+    return 8
+}
+
+
+/// decode compact size from vector
+public fun compact_size(v: vector<u8>, start: u64): (u256, u64) {
```

Comment:
wont u64 be enough as the first return value?

Reply:
Yes, this is enough for the first value. However to_to_number return u256 type. 
Generic type may be useful in this case.